application/src/main/resources/db/V11__Add_Help_Channel_Plus_Creator.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE help_threads
```

Comment:
i dont like the name of this migration file. its too specific and easily outdated. rename to `...Add_Help_Thread_Metadata...` maybe